examples/mobile/UIComponents/components/dialogs/dialog.html

```diff
@@ -348,16 +334,35 @@ <h1>
 					Dialog header
 				</div>
 				<div class="ui-popup-content">
-					<div class="ui-popup-controlbox">
-						<div><input type="radio" name="test"/>List</div>
-						<div><input type="radio" name="test"/>Detailed list</div>
-						<div><input type="radio" name="test"/>Grid</div>
-					</div>
+					<ul class="ui-listview ui-content-area">
+						<li class="ui-li-has-radio">
+							<input checked="checked" id="radio-1" name="radio-group" type="radio" />
```

Comment:
By default, it look "List" has the "checked" value.
But in the test, it don't display the default check.
Would you check it, please?

Reply:
checked.

done.